src/main/java/net/sf/jabref/importer/fileformat/BibtexParser.java

```diff
@@ -283,7 +262,7 @@ public ParserResult parse() throws IOException {
                                     + be.getAuthorTitleYear(40) + " ("
                                     + Localization.lang("grouping may not work for this entry") + ")");
                         }
-                    } catch (IOException ex) {
+                    } catch (Exception ex) {
```

Comment:
why did you change to Exception here? The code above does not state that Exception is required.